tests/OSPSuite.R.Tests/Services/ContainerTaskSpecs.cs

```diff
@@ -165,8 +169,8 @@ public class When_resolving_all_parameters_recursively_from_a_container : concer
       [Observation]
       public void should_return_the_matching_parameters()
       {
-         sut.AllParametersMatching(_organism, Constants.WILD_CARD_RECURSIVE).ShouldOnlyContain(_organism.GetAllChildren<IParameter>());
```

Comment:
This is not ok anymore
** is meant to replace tree structure. so if I Want everything it should be **|* e.g. all possible level and all names). 
I think this was a mistake